app/models/movie.rb

```diff
@@ -3,5 +3,6 @@ class Movie < ApplicationRecord
   has_many :directors, through: :directions
 
   validates :original_title, presence: true
+  validates :year, presence: true
```

Comment:
Should we also validate the presence of the TMDb id?

If so, we should also add it to the spec.